### PR TITLE
Fixed add liquidity for first, second or any number of times

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Closes - https://centralitydev.atlassian.net/browse/CORE-41

### CODE REVIEW

File: src/pages/exchange/index.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
+import {connect} from 'react-redux';
```
Comment (KarishmaBothara) [inline src/pages/exchange/index.tsx:1]: Please ignore this file related to exchange/index.tsx
It was accidentally deleted and re-introduced

**Comment by holygits** [comment]
Screen Shot 2021-03-29 at 13 05 41

"Transaction fee 1.378 converted to 0.3480 CPAY" can we make it "Transaction fee (estimated): 0.3480 CPAY"

Screen Shot 2021-03-29 at 13 07 43
remove quotes here

code changes look ok so far, need to setup custom extension to test more

**Comment by KarishmaBothara** [comment]
This PR is ready for review now

File: src/redux/epics/liquidity/liquidityRate.epic.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+import types from '../../actions';
+import {
+    requestLiquidityPrice,
+    requestLiquidityValue,
+    updateAddAsset1Amount,
+    updateAddAsset2Amount,
+} from '../../actions/ui/liquidity.action';
+import {AppState} from '../../reducers';
+
+export const getCoreLiquidityValueEpic = (
```
Comment (holygits) [inline src/redux/epics/liquidity/liquidityRate.epic.ts:18]:
```suggestion
export const getCoreLiquidityPriceEpic = (
```

File: src/redux/epics/liquidity/liquidityRate.epic.ts [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    action$: Observable<Action<any>>,
+    store$: Observable<AppState>,
+    {api$}: IEpicDependency
+) =>
+    combineLatest([api$, action$.pipe(ofType(types.ui.Liquidity.EXCHANGE_LIQUIDITY_VALUE))]).pipe(
```
Comment (holygits) [inline src/redux/epics/liquidity/liquidityRate.epic.ts:23]:
```suggestion
    combineLatest([api$, action$.pipe(ofType(types.ui.Liquidity.EXCHANGE_CORE_LIQUIDITY_PRICE))]).pipe(
```

**Review by holygits — COMMENTED** [review]
lgtm. if its possible to add a couple test cases for the new epics that would be great

File: src/redux/epics/global.epic.ts [hunk 4]
```diff
@@ -50,9 +50,6 @@ const getExtensionMetadata = (
                             api.runtimeVersion.specVersion
                         );
                         // remove all the classes from the spectypes
-                        // The reason for the removal of the following object from specTypes -
```
[the remark this answers is not in the document]
Comment (KarishmaBothara) [reply src/redux/epics/global.epic.ts:53]: no.. will add it back.. resolving conflict resulted in this

**Review by holygits — APPROVED** [review]

**Pull request merged**